lib/charge.js

```diff
@@ -15,6 +15,10 @@ if (argv.version) {
   process.exit(0)
 }
 
+if (argv.pageSize && argv.pageSize[0] === '{') {
```

Comment:
I noticed while merging some of my own pull requests that having this in charge.js will work fine if electron-pdf is invoked using the CLI.  But if you want to use it as a module in a program then charge.js is not used.  This should be moved somewhere inside index.js.  

Maybe you can do that when you write some tests?